### PR TITLE
update gradle file 

### PR DESCRIPTION
**Pull request by weijjia**
update gradle files for release:
1. update version to reflect debug and release version
2. add task to generate pom and source.jar

### CODE REVIEW

File: msal/build.gradle [hunk 1]
```diff
@@ -2,6 +2,12 @@ apply plugin: 'com.android.library'
 apply plugin: 'jacoco'
 apply plugin: 'checkstyle'
 apply plugin: 'pmd'
+apply plugin: "org.ajoberstar.grgit"
+// This plugin publishes adal in to the local maven repo
```
Comment (iambmelt) [inline msal/build.gradle]: msal, not adal

**Comment by iambmelt** [comment]
Do we have deployment/publishing steps documented anywhere? It looks like this process begins with a `./gradlew createpom` but what happens after that is unclear to me

File: msal/build.gradle [hunk 2]
```diff
@@ -82,6 +101,50 @@ task javadocJar(type: Jar, dependsOn: javadoc) {
...
+                    name 'Microsoft'
+                }
+            }
+
+            licenses {
+                license {
+                    name 'Apache License, Version 2.0'
+                }
+            }
+            inceptionYear '2014'
```
Comment (iambmelt) [inline msal/build.gradle]: This appears outdated

File: msal/build.gradle (hunk 1 above)
Comment (weijjia) [inline msal/build.gradle]: done.

**Comment by weijjia** [comment]
createPom task will generate the pom file

**Pull request merged**